docs/reference/elasticsearch/configuration-reference/elasticsearch-settings.md

```diff
@@ -0,0 +1,148 @@
+
+Elasticsearch Service automatically rejects `elasticsearch.yml` settings that could break your cluster. For a list of supported settings, check [Supported {{es}} settings](index.md).
+
+::::{tip}
+Some settings that could break your cluster if set incorrectly are blocked, such as certain zen discovery and security settings. For examples of a few of the settings that are generally safe in cloud environments, check [Edit stack settings](docs-content://deploy-manage/deploy/cloud-enterprise/edit-stack-settings.md) for {{ece}} and  [Edit stack settings](docs-content://deploy-manage/deploy/elastic-cloud/edit-stack-settings.md) for the {{ech}} offering.
+::::
+
+::::{warning}
+You can also update [dynamic cluster settings](docs-content://deploy-manage/deploy/self-managed/configure-elasticsearch.md#dynamic-cluster-setting) using {{es}}'s [update cluster settings API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-cluster-put-settings). However, Elasticsearch Service doesn’t reject unsafe setting changes made using this API. Use with caution.
+::::
+
+## Add or edit user settings for {{ech}}:
+```{applies_to}
+  deployment:
+    ess: 
+```
+
+1. Log in to the [Elasticsearch Service Console](https://cloud.elastic.co?page=docs&placement=docs-body).
+2. Find your deployment on the home page in the Elasticsearch Service card and select **Manage** to access it directly. Or, select **Hosted deployments** to go to the deployments page to view all of your deployments.
+
+    On the deployments page you can narrow your deployments by name, ID, or choose from several other filters. To customize your view, use a combination of filters, or change the format from a grid to a list.
+
+3. From your deployment menu, go to the **Edit** page.
+4. In the **Elasticsearch** section, select **Manage user settings and extensions**.
+5. Update the user settings.
+6. Select **Save changes**.
+
+::::{note}
+In some cases, you may get a warning saying "User settings are different across Elasticsearch instances". To fix this issue, ensure that your user settings (including the comments sections and whitespaces) are identical across all Elasticsearch nodes (not only the data tiers, but also the Master, Machine Learning, and Coordinating nodes).
+::::
+
+## Supported {{es}} settings [ec-es-elasticsearch-settings]
+
+Elasticsearch Service supports the following `elasticsearch.yml` settings.
+
+### General settings [ec_general_settings]
+
+`xpack.notification.webhook.additional_token_enabled`
+:   When set to `true`, {{es}} automatically sets a token which enables the bypassing of traffic filters for calls initiated by Watcher towards {{es}} or {{kib}}. The default is `false` and the feature is available starting with {{es}} version 8.7.1 and later.
+
+    ::::{important}
+    This setting only applies to the Watcher `webhook` action, not the `http` input action.
+    ::::
+
+
+### X-Pack [ec_x_pack]
```

Comment:
> ### X-Pack [ec_x_pack]

This is an old term and I don't think it makes sense to carry it forward into these new docs.